lib/membrane/bin.ex

```diff
@@ -174,9 +165,15 @@ defmodule Membrane.Bin do
               state :: state_t
             ) :: callback_return_t
 
+  @doc """
+  Callback invoked when bin is removed by its parent.
```

Comment:
I'm not 100% sure, but I'd say:
```suggestion
  A callback invoked when the bin is being removed by its parent.
```